src/pages/edge/edge-network/tabs/api-reference.md

```diff
@@ -235,16 +236,94 @@ Edge.sendEvent(experienceEvent, null);
 Edge.sendEvent(experienceEvent, new EdgeCallback() {
   @Override
   public void onComplete(final List<EdgeEventHandle> handles) {
-        // Handle the Edge Network response 
+        // Handle the Edge Network response
+  }
+});
+```
+
+##### Example with Datastream ID override
+
+```java
+// Create experience event from Map
+Map<String, Object> xdmData = new HashMap<>();
+xdmData.put("eventType", "SampleXDMEvent");
+xdmData.put("sample", "data");
+
+ExperienceEvent experienceEvent = new ExperienceEvent.Builder()
+  .setXdmSchema(xdmData)
+  .setDatastreamIdOverride("SampleDatastreamId")
+  .build();
+```
+
+```java
+// Example 1 - send the experience event without handling the Edge Network response
+Edge.sendEvent(experienceEvent, null);
+```
+
+```java
+// Example 2 - send the experience event and handle the Edge Network response onComplete
+Edge.sendEvent(experienceEvent, new EdgeCallback() {
+  @Override
+  public void onComplete(final List<EdgeEventHandle> handles) {
+    // Handle the Edge Network response
   }
 });
```

Comment:
Can we club these together in the same code block where you create the event and sent it, and probably only one option (with/without handle) is good enough for this example.
This comment applies for all similar examples.